### ISSUE TITLE
the model doesn't overfit

### ISSUE THREAD

**Issue by zghn** [kevinlin311tw/keras-openpose-reproduce#18]
Hi 
I'm trying to overfit the model on 10 images but the loss and accuracy do not change a lot or sometimes loss gets nan!  what's wrong with it? would you help me to find out the right processes?
Before it I've tried to train the model but I couldn't. so I decided to make it overfitted.
